### ISSUE TITLE
Docker - creation of tables ?

### ISSUE THREAD

**Issue by remileduc** [eark-project/earkweb#41]
When I follow you instructions to setup earkweb through docker (https://github.com/eark-project/earkweb/blob/master/docs/install_docker.md), the eark database is correctly created, but it doesn't have any tables... So everything crashes afterwards (can't create user...)

When I run `docker exec -it --user=root earkdb_1 /repair_tables.sh` it tells me that the files doesn't exist (`myisamchk: error: File '/var/lib/mysql/eark/auth_group' doesn't exist...`), I don't know how to get them :/

also, it seems that no containers has the urllib3

``` python
celery_1    |   File "/earkweb/sip2aip/views.py", line 9, in <module>
celery_1    |     from requests.packages.urllib3.exceptions import ConnectionError
celery_1    | ImportError: No module named packages.urllib3.exceptions
celery_1 exited with code 1
```

Have I done something wrong? how could I solve it? Is there an easy way to setup earkweb?
I can't find any definition of the SQL tables in your github...

**Comment by shsdev** [comment]
It seems the database was not created properly.

The docker setup is done by the following script:

https://github.com/eark-project/earkweb/blob/master/docker-compose-run.sh

In this script, the database is created by calling the following script:

https://github.com/eark-project/earkweb/blob/master/docker/earkdb/init.sh

and the SQL tables are created automatically by Django (data model) if they do not exist, that is why there is not SQL setup file.

Regarding the missing pyhton module urllib3, we will have to check if it needs to be added.

**Comment by shsdev** [comment]
Because of the urllib3 dependency the Docker set-up did not function properly.

Docker installation is now fixed with commit e3694db09499e7d6b4d5ba3fedbb0d6bafc968fd

Furthermore, images for earkweb and the database are now provided on Docker Hub. This speeds up installation which is now possible in a few minutes if docker-engine and docker-compose are installed on the host system. 

The docker documentation is updated accordingly.
